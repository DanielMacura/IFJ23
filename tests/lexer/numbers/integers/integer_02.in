let x = 123
x = 456
x = 789