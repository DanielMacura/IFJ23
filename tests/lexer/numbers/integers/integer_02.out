KW_LET
VARIABLE
EQUALS
INTEGER:123
EOL
VARIABLE
EQUALS
INTEGER:456
EOL
VARIABLE
EQUALS
INTEGER:789
EOF
